### ISSUE TITLE
Decompile PyCodeObject from CPython

### ISSUE THREAD

**Issue by iamdavidcz** [zrax/pycdc#192]
Hello,

Is there an API to programmatically decompile PyCodeObject from CPython?

Regards,
Dawid

**Comment by zrax** [comment]
No, there's no API or library currently, let alone CPython bindings; the source is just compiled into the two executables (pycdc and pycdas).  I suppose you could just write the .pyc headers and call pycdc from a `subprocess` though...
